### PR TITLE
Dev server crash fix

### PR DESCRIPTION
**Pull request by ghost**
Fix for #199.

Checks that the request returned a 200. API fails return 400/401.

**THIS NEEDS TO BE TESTED WITH REAL API KEYS. I don't have any.**

### CODE REVIEW

File: controllers/resources.js [hunk 1]
```diff
@@ -147,13 +147,13 @@ module.exports = {
...
-            trello = trello ? (JSON.parse(trello)).length : "Can't connect to to Trello";
+            trello = (status && status.statusCode == 200) ? (JSON.parse(trello)).length : "Can't connect to to Trello";
             res.send({"trello": trello});
         });
     },
     bloggerCalls: function(req, res) {
         request('https://www.googleapis.com/blogger/v3/blogs/2421288658305323950/posts?key=' + secrets.blogger.key, function (err, status, blog) {
-            blog = blog.length > 100 ? JSON.parse(blog) : '';
+            blog = (status && status.statusCode == 200) ? JSON.parse(blog) : '';
             res.send({
```
Comment (BerkeleyTrue) [inline controllers/resources.js]: @jameskopacz Thanks! Can you add `err` handling here? Basically, add `next` as the third argument to the outer function (i.e. `req, res, next`) and above the lines you modified add `if (err) { return next(err); }`

**Comment by jameskopacz** [comment]
Like that?

**Comment by BerkeleyTrue** [comment]
Sweet! Looks good. Thanks

**Comment by BerkeleyTrue** [comment]
Sorry haven't got around to this yet. Will do by the end of the day.

**Comment by QuincyLarson** [comment]
@BerkeleyTrue Are you still QA'ing this?

**Comment by BerkeleyTrue** [comment]
Sorry dude. I have been on a support ticket for a client all day. Can you take this one?

**Comment by BerkeleyTrue** [comment]
on it now.

**Comment by BerkeleyTrue** [comment]
Works. There are a bunch more in this file that need to be handled properly. Will merge though as these work.

**Pull request merged**